packages/grafana-ui/src/components/Tabs/Tab.tsx

```diff
@@ -38,18 +38,20 @@ export const Tab = React.forwardRef<HTMLAnchorElement, TabProps>(
     const linkClass = cx(tabsStyles.link, active ? tabsStyles.activeStyle : tabsStyles.notActive);
 
     return (
-      <li className={tabsStyles.item}>
+      <div className={tabsStyles.item}>
```

Comment:
I hope this change is fine. With `ul` in the parent component and `li` here I couldn't get a11y right: pa11y would complain that `role=tablist` didn't contain any `role=tab` children. It seems as it requires `role=tab` to be an immediate child of `role=tablist` when `ul`/`li` is used 🤷 

I needed `role=tab` + `aria-selected` to figure out if the correct tab is selected in the integration test.

Reply:
i think changing from `ul/li` to divs is fine, it shouldn't degrade a11y (if any, you changes will improve it).

as mentioned offline, making tabs/tablist actually accessible would be a bigger (and more involved) project, so i'm fine with the change 👍